petric.py

```diff
@@ -88,26 +88,51 @@ def __call__(self, algo: Algorithm):
         log.debug("...logged")
 
 
+class QualityMetrics(ImageQualityCallback):
+    """From https://github.com/SyneRBI/PETRIC/wiki#metrics-and-thresholds"""
+    def __init__(self, reference_image, whole_object_mask, background_mask, **kwargs):
+        super().__init__(reference_image, **kwargs)
+        self.whole_object_indices = np.where(whole_object_mask == 1)
+        self.background_indices = np.where(background_mask == 1)
+        self.ref_im_arr = reference_image.as_array()
+
+    def __call__(self, algorithm):
+        iteration = algorithm.iteration
+        if iteration % algorithm.update_objective_interval != 0 and iteration != algorithm.max_iteration:
```

Comment:
Is this a good idea? I can imagine we'd want to compute the metrics every update for the final evaluation, while the obj-func less frequently. Maybe introduce an optional arg `update_interval`, if set to `None`, use `algorithm.update_objective_interval`

Reply:
will do in a separate issue/PR (probably upstream)

Reply:
moved to #50